### PR TITLE
Car docs: dashcam and community support docs

### PR DESCRIPTION
**Pull request by sshane**
- [x] support generating list of files

### CODE REVIEW

File: docs/PORTS.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+    * 2016/2017 Accord
+    * 15-20 Acura MDX (2022+ MDX does not use serial steering and uses the same Bosch harness, but the wiring pinout is completely different)
+    * 15-20 Acura TLX (2021+ TLX uses the same Bosch harness with the same wiring pinout)
+
+    Join the #topic-serial-steering channel on discord with questions.
+
+
+
+# Don't see your car here?
```
Comment (adeebshihadeh) [inline docs/PORTS.md:79]: FYI, think we link to this on web3

Comment (sshane) [reply docs/PORTS.md]: We already do link to this section, on CARS.md.

**Comment by sshane** [comment]
@adeebshihadeh anything else you want for now?

File: docs/PORTS.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+|Chevrolet|Malibu Premier 2017[<sup>4</sup>](#footnotes)|Adaptive Cruise Control (ACC)|openpilot|18 mph|7 mph|[![star](assets/icon-star-full.svg)](##)|[![star](assets/icon-star-empty.svg)](##)|<details><summary>Parts</summary><sub>- 1 OBD-II connector<br>- 1 comma 3X<br>- 2 long OBD-C cable<br>- 1 mount<br>- 1 right angle OBD-C cable (1.5 ft)<br><a href="https://comma.ai/shop/comma-3x.html?make=Chevrolet&model=Malibu Premier 2017">Buy Here</a></sub></details>||
+|Ford|F-150 2023|Co-Pilot360 Active 2.0|openpilot available[<sup>1</sup>](#footnotes)|0 mph|0 mph|[![star](assets/icon-star-full.svg)](##)|[![star](assets/icon-star-full.svg)](##)|<details><summary>Parts</summary><sub>- 1 Ford Q3 connector<br>- 1 RJ45 cable (7 ft)<br>- 1 comma 3X<br>- 1 comma power v2<br>- 1 harness box<br>- 1 mount<br>- 1 right angle OBD-C cable (1.5 ft)<br><a href="https://comma.ai/shop/comma-3x.html?make=Ford&model=F-150 2023">Buy Here</a></sub></details>||
```
Comment (rwxr-xr-x) [inline docs/PORTS.md:15]: I'm an 2023 F-150 XLT with Co-Pilot360 Assist 2.0 owner - see below what the package consists of for a 2023 F-150, you can ignore this comment is you this it's not appropriate. 

image

I recommend the following changes:

Change `Co-Pilot360 Active 2.0` to `Co-Pilot360 Assist 2.0`
Change `1 Ford Q3 connector` to `1 Ford Q4 connector`
Change `1 RJ45 cable (7 ft)` to `1 RJ45 cable (1 ft) - or what comes with comma 3X`
Change `1 comma power v2` to `1 comma power v2 (10 ft USB-C extension cable)` - the IPMA is under the steering wheel, close to the ODB-II port.
Change `1 mount` to `1 8degree mount`

in other words I ordered:
- C3X + Q4 harness from comma
- 3D printed a 8degree mount
- ordered a 1ft Ethernet cable and a 10ft USB-C extension cable, maybe a 6.6 ft would be enough but better safe than sorry.

Comment (sshane) [reply docs/PORTS.md:15]: Can you open a pull request for these changes, if they're more correct for the US market?

**Comment by sshane** [comment]
@adeebshihadeh LGTM?

File: docs/PORTS.md (hunk 2 above)
Comment (rwxr-xr-x) [reply docs/PORTS.md:15]: Sure, I can create one, once you have your changes merged. If there is a better option, let me know.

**Comment by adeebshihadeh** [comment]
Let's start with https://github.com/orgs/commaai/projects/28
